pfcpiface/node.go

```diff
@@ -51,10 +51,35 @@ func NewPFCPNode(ctx context.Context, upf *upf) *PFCPNode {
 	}
 }
 
+func (node *PFCPNode) tryConnectToN4Peer(lAddrStr string) {
```

Comment:
Since this function iterates over peers it's better to name it `tryConnectToN4Peers` instead.